homeassistant/components/unifiprotect/const.py

```diff
@@ -0,0 +1,54 @@
+"""Constant definitions for UniFi Protect Integration."""
+
+# from typing_extensions import Required
+from datetime import timedelta
+
+from pyunifiprotect.data.types import ModelType, Version
+
+DOMAIN = "unifiprotect"
+
+ATTR_EVENT_SCORE = "event_score"
+ATTR_EVENT_OBJECT = "event_object"
+ATTR_EVENT_THUMB = "event_thumbnail"
+ATTR_WIDTH = "width"
+ATTR_HEIGHT = "height"
+ATTR_FPS = "fps"
+ATTR_BITRATE = "bitrate"
+ATTR_CHANNEL_ID = "channel_id"
+
+CONF_DOORBELL_TEXT = "doorbell_text"
```

Comment:
Many constants here are never used. Please don't add any constants until they are actually used even if it's planned later.

Reply:
Sorry about that, I thought I cleaned them all out. Migrating the HACS integration in chunks kind of leads to me not getting _everything_.